### PR TITLE
ramips: rework network settings for HC5861

### PR DESCRIPTION
**Pull request by ptpt52**
dts: disable port4 and leave it ephy mode because it connect to nothing
switch port5 connected to GE port we use it as wan port

### CODE REVIEW

**Comment by ptpt52** [comment]
HC5861:
2 FE port + 1 GE port
2 lan + 1 wan

File: target/linux/ramips/dts/HC5861.dts [hunk 1]
```diff
@@ -90,30 +84,29 @@
...
+		compatible = "pci14c3,7662";
 		reg = <0x0000 0 0 0 0>;
 		mediatek,mtd-eeprom = <&factory 0x8000>;
 		ieee80211-freq-limit = <5000000 6000000>;
 	};
 };
+
+&state_default {
+	gpio {
+		ralink,group = "uartf", "wled", "rgmii2", "ephy";
```
Comment (981213) [inline target/linux/ramips/dts/HC5861.dts]: It seems that this comment is too late as this got merged already. :(
This router only used uartf pins so you only need uartf here. There is no need to set unused pinmux groups in dts.

Comment (981213) [reply target/linux/ramips/dts/HC5861.dts]: And uartf is already added in HC5X61.dtsi so this pinmux node can be dropped.

Comment (ptpt52) [reply target/linux/ramips/dts/HC5861.dts]: I don't know what it means, I thought that if these are not used for pinctrl, they should be put in the gpio group.

Anyway, it seems that there is no problem?

Comment (981213) [reply target/linux/ramips/dts/HC5861.dts]: @ptpt52 I might not able to explain it clear enough in English so I've sent some Chinese explanation to your mail :)

**Comment by ptpt52** [comment]
I push addition commit:
ramips: HC5861 drop unused gpio group define in dts

for some reason the previous commit has been merged, I rebase and add a new change to fix the DTS

**Comment by chunkeey** [comment]
Merged. Thanks.

**Pull request closed without merging**